src/s1reader/s1_burst_slc.py

```diff
@@ -652,8 +652,9 @@ def eap_compensation_lut(self):
                             f' IPF version = {self.ipf_version}')
 
         return self.burst_eap.compute_eap_compensation_lut(self.width)
-    def bbox(self):
```

Comment:
I removed this once I realized why the `border` was a list of Polygon's, when it seemed like it was always a list of one single Polygon: https://github.com/opera-adt/burst_db/issues/1

the anti-meridian crossing bursts (international date line) have two polygons, since it's conventional to split the latlon that way, so this function is giving an incorrect bbox for those ones. I can leave it in and submit an issue to correct it in the future if you'd like.

Reply:
fixing it might be as simple as 
```
    return shapely.geometry.MultiPolygon(b.border).bounds
```
as long as we don't care that it returns longitudes greater than 180:
```python
In [5]: b = s1reader.load_bursts('S1A_IW_SLC__1SDV_20220110T181913_20220110T181938_041402_04EC40_E2D9.zip', None, 1)[0]
In [6]: b.border
Out[6]: [<shapely.geometry.polygon.Polygon object at 0x7fadef947670>, <shapely.geometry.polygon.Polygon object at 0x7fadef944d00>]

In [8]: b.border[0].bounds
Out[8]: (180.0, 52.19675210021637, 180.6997402722514, 52.44622930532407)

In [11]: import shapely.geometry
In [12]: shapely.geometry.MultiPolygon(b.border).bounds
Out[12]: (179.3998234765543, 52.19675210021637, 180.6997402722514, 52.50961732430616)

```

Reply:
Let's submit an issue separately. We do not want to have longitudes greater than 180 (if this is the only fix)